Source/Requests/HVACRequest.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+public struct HVACReqeust: RequestProtocol {
```

Comment:
I think the name of this `struct` contains a typo 🙃

Reply:
Whoops, I will fix that asap.